### PR TITLE
feat(lcd): allow custom color schemes

### PR DESCRIPTION
**Pull request by doinkythederp**

### CODE REVIEW

File: pros/src/sensors/vision.rs
Comment (Gavin-Niederman) [inline pros/src/sensors/vision.rs:1]: What is this? Why is there stuff about the lcd in this file?

Comment (doinkythederp) [reply pros/src/sensors/vision.rs:1]: Oh, I was going to add conversions between Rgb and LcdColor but forgot to finish.

File: pros/src/lvgl/colors.rs
Comment (Gavin-Niederman) [inline pros/src/lvgl/colors.rs:1]: Maybe you could document what color space LcdColor is.

**Review by Gavin-Niederman — APPROVED** [review]
looks good!

**Pull request merged**